chain/db/statedb.go

```diff
@@ -10,12 +10,13 @@ import (
 )
 
 type account struct {
-	nonce   uint64
-	balance common.Fixed64
+	nonce               uint64
+	balance             common.Fixed64
+	nanoPaymentChannels map[string]common.Fixed64
```

Comment:
Map is a unordered type. So this will make the serialization unequal.

Reply:
you mean if I serialize(1)+deserialize+serialize(2) then byte contents in 1 and 2 will be different? why does it matter though?

Reply:
This is a requirement because the state root is stored in block, so the serialization of world state needs to be bit-wise identical on all nodes, otherwise block proposed by one node may be considered invalid by other nodes because they got a different state root.

Reply:
ok, fixed